miso-web/src/main/webapp/scripts/pool_ajax.js

```diff
@@ -830,4 +830,87 @@ Pool.barcode = {
       }
     );
   }
-};
\ No newline at end of file
+};
+
+Pool.orders = Pool.orders || {
+  'makeXhrRequest': function (method, endpoint, callback, data, callbackarg) {
+    var expectedStatus;
+    var unauthorizedStatus = 401;
+    if (method == 'POST') {
+      expectedStatus = [201, 200]; // will be 200 in case of a POST to sampleclass which triggers a GET for samplevalidrelationship
+    } else {
+      expectedStatus = [200, 404];
```

Comment:
404 is no longer expected since querying an empty collection returns a 200 with empty array (which reminds me, I need to make some changes to other files...)